### PR TITLE
Renaming (Performance | Dependencies) workbooks and categories

### PR DESCRIPTION
**Pull request by nasundar**
1. Renamed performance and dependencies workbooks to follow a pattern-
    {Category} for a {Type}

  Category: Network Dependencies | Performance Analysis
  Type: Single VM | Group of VMs
2. Moved Single VM Performance Analysis to 'Performance Analysis' category

### CODE REVIEW

File: Workbooks/Log Analytics - Network Dependencies/categoryResources.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "en-us": {"name":"Network Dependencies", "description": "Look for network dependencies of your Virtual Machines.", "order": 200}
+}
```
Comment (andrewki-msft) [inline Workbooks/Log Analytics - Network Dependencies/categoryResources.json]: Would you add an empty newline at the end of every JSON file?

File: Workbooks/Log Analytics - Performance Analysis/Performance Analysis for a group of VMs/settings.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
-    "name":"VM Performance Analysis",
+    "name":"Performance Analysis for a group of VMs",
```
Comment (vutran01) [inline Workbooks/Log Analytics - Performance Analysis/Performance Analysis for a group of VMs/settings.json]: should group be upper case to match with above?

File: Workbooks/Virtual Machines - Performance Analysis/categoryResources.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "en-us": {"name":"Performance Analysis", "description": "Look for common issues with your VM via metrics", "order": 200}
```
Comment (vutran01) [inline Workbooks/Virtual Machines - Performance Analysis/categoryResources.json]: VMs?

File: Workbooks/Log Analytics - Performance Analysis/Performance Analysis for a group of VMs/settings.json (hunk 2 above)
Comment (andrewki-msft) [reply Workbooks/Log Analytics - Performance Analysis/Performance Analysis for a group of VMs/settings.json]: Would you also rename the file to match as well?

Comment (nasundar) [reply Workbooks/Log Analytics - Performance Analysis/Performance Analysis for a group of VMs/settings.json]: Thanks.

File: Workbooks/Log Analytics - Network Dependencies/categoryResources.json (hunk 1 above)
Comment (nasundar) [reply Workbooks/Log Analytics - Network Dependencies/categoryResources.json]: Thanks.

**Pull request merged**